test/binary/suite.ini

```diff
@@ -2,3 +2,4 @@
 core = tarantool
 script = binary.lua
 description = memcached binary tests
+disable = binary-boundary.test.py
```

Comment:
Looks like this doesn't disable test properly https://github.com/tarantool/memcached/runs/4397060010?check_suite_focus=true#step:8:61

Reply:
I have updated PR, so it is not relevant.